src/cgame/cg_draw.cpp

```diff
@@ -890,6 +985,51 @@ static void CG_DrawLagometer() {
     CG_DrawBigString(ax, ay, "snc", 1.0);
   }
 
+  // snapshot display
+  vec4_t textColor = {0.625f, 0.625f, 0.6f, 1.0f};
+  float fps;
+
+  if (cg.demoPlayback) {
+    static bool svFpsAvailable =
+        ETJump::demoCompatibility->isCompatible({3, 3, 0});
+    static bool csAvailable =
+        ETJump::demoCompatibility->isCompatible({3, 2, 0});
+
+    if (svFpsAvailable) {
```

Comment:
Could this whole piece of code be moved to democompat, e.g. `ETJump::demoCompatibility->getSvFps()`

Reply:
Done, moved it to `etj_utilities.cpp` instead as it's likely going to be used in the future a lot with pmove related drawing stuff, to determine if we can safely use pmove command time for timing intervals.